i18n/zh/docusaurus-plugin-content-docs/current/reference/components/karmada-agent.md

```diff
@@ -2,19 +2,16 @@
 title: karmada-agent
 ---
 
+### 简介
```

Comment:
pls update the first headtag `###` to `##`, also english version.

Reply:
Done， and thanks for review